documentation/docs/20-core-concepts/10-routing.md

```diff
@@ -12,6 +12,19 @@ At the heart of SvelteKit is a _filesystem-based router_. The routes of your app
 
 Each route directory contains one or more _route files_, which can be identified by their `+` prefix.
 
+Summary of the routing files:
+
+| File name         | Role                                                                | Propagates to sub-dirs | SSR | CSR |
+|-------------------|---------------------------------------------------------------------|:----------------------:|:---:|:---:|
+| +page.svelte      | Create an HTML page                                                 |           ❌            |  ✅  |  ✅  |
+| +error.svelte     | Create an HTML page to render HTTP errors                           |           ✅            |  ✅  |  ✅  |
+| +layout.svelte    | Create an HTML layout                                               |           ✅            |  ✅  |  ✅  |
+| +page.js          | Provide data for the **page** via the `load()` function             |           ❌            |  ✅  |  ✅  |
```

Comment:
These should have an "s" at the end. E.g. you would say "+page.svelte creates an HTML page"
```suggestion
| +page.svelte      | Creates an HTML page                                                |           ❌            |  ✅  |  ✅  |
| +error.svelte     | Creates an HTML page to render HTTP errors                          |           ✅            |  ✅  |  ✅  |
| +layout.svelte    | Creates an HTML layout                                              |           ✅            |  ✅  |  ✅  |
| +page.js          | Provides data for the **page** via the `load()` function            |           ❌            |  ✅  |  ✅  |
```